src/background.js

```diff
@@ -829,13 +829,13 @@ function onMessageHandler(request, sender, callback) {
 				if (!Array.isArray(sub)) {
 					sub = [sub];
 				}
-				const subscriptionData = sub.reduce((acc, curr) => {
-					let a = acc;
-					if (curr.productName.includes('Plus')) {
-						a = curr;
-					}
-					return a;
-				}, {});
+
+				let subscriptionData = sub.find(subscription => subscription.productName.includes('Ghostery Premium'));
+				if (subscriptionData) {
+					callback({ subscriptionData });
+				}
+
+				subscriptionData = sub.find(subscription => subscription.productName.includes('Ghostery Plus'));
 				callback({ subscriptionData });
```

Comment:
See my comment on `Account#getUserSubscriptionData`. Note that even if we don't make that change, we need to make sure we pass an empty object to the callback, and not `undefined` (the fail return value of `Array.prototype.find`), in the event that neither subscription is found.

Reply:
Changed to return only the highest tier subscription, and made sure to return an empty object instead of `undefined`!